stores/sql/mysql/migrations/main/migration_00018_idx_db_directory.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS `idx_objects_db_directory_id` ON `objects`(`db_directory_id`);
```

Comment:
I think you put this in the wrong file? The index is missing in SQLite, so you should migrate it there. I would also add a comment to the empty file with something like `-- idx_objects_db_directory_id was only missing in SQLite` or something to explain the empty file.